oauth2_provider/tests/test_application_views.py

```diff
@@ -76,6 +77,12 @@ def setUp(self):
         self.app_bar_1 = self._create_application('app bar_user 1', self.bar_user)
         self.app_bar_2 = self._create_application('app bar_user 2', self.bar_user)
 
+    def test_get_application(self):
+        """Test the get_application method from the AuthorizationView"""
```

Comment:
It would be helpful if the docstring explained what about the method you are testing. For example, `Verify the method returns the application corresponding to the given client ID.`